src/sentry/testutils/relay.py

```diff
@@ -158,6 +158,8 @@ def relay_setup_fixtures(
         self,
         settings,
         live_server,
+        default_project,
```

Comment:
Those are needed, as we cannot use `BaseTest` anymore, which inherited those fixtures.